### PR TITLE
Add nn::oe::TryPopLaunchParameter

### PR DESCRIPTION
**Pull request by ecumber**
Adds nn::oe::TryPopLaunchParameter, needed by nxargs in BOTW

---
This change is [Reviewable](https://reviewable.io/reviews/open-ead/nnheaders/8)

### CODE REVIEW

**Comment by ecumber** [comment]
Do not merge yet, I have to figure out the types so it can be used in BOTW

File: include/nn/oe.h [hunk 1]
```diff
@@ -43,5 +43,6 @@ void SetUserInactivityDetectionTimeExtended(bool);
 void FinishStartupLogo();
 nn::settings::LanguageCode GetDesiredLanguage();
 void GetDisplayVersion(DisplayVersion*);
+bool TryPopLaunchParameter(void*, u64); // TODO : figure out types for this
```
Comment (leoetlino) [inline include/nn/oe.h:46]: That's not the correct signature. This should be:

    nn::oe::TryPopLaunchParameter(ulong *,void *,ulong)

according to the symbol, which is probably `nn::oe::TryPopLaunchParameter(size_t*,void *,size_t)`

**Review by leoetlino — DISMISSED** [review]

File: include/nn/oe.h [hunk 2]
```diff
@@ -43,5 +43,6 @@ void SetUserInactivityDetectionTimeExtended(bool);
 void FinishStartupLogo();
 nn::settings::LanguageCode GetDesiredLanguage();
 void GetDisplayVersion(DisplayVersion*);
+bool TryPopLaunchParameter(size_t*,void*,size_t);
```
Comment (leoetlino) [inline include/nn/oe.h:46]: formatting
```suggestion
bool TryPopLaunchParameter(size_t* ,void*, size_t);
```

File: include/nn/oe.h [hunk 3]
```diff
@@ -43,5 +43,6 @@ void SetUserInactivityDetectionTimeExtended(bool);
 void FinishStartupLogo();
 nn::settings::LanguageCode GetDesiredLanguage();
 void GetDisplayVersion(DisplayVersion*);
+bool TryPopLaunchParameter(size_t* ,void*, size_t);
```
Comment (leoetlino) [inline include/nn/oe.h:46]: Sorry I messed up the comma in my comment. (If you find a review suggestion weird feel free to ask questions)

```suggestion
bool TryPopLaunchParameter(size_t*, void*, size_t);
```

**Pull request merged**